pkg/sink/interceptors_test.go

```diff
@@ -88,7 +88,7 @@ func TestCreateOutgoingRequest(t *testing.T) {
 		"foo":       "bar",
 	})
 
-	req, _ := http.NewRequest(http.MethodPost, "http://event.listener.url", ioutil.NopCloser(bytes.NewBuffer(reqBody)))
+	req := httptest.NewRequest(http.MethodPost, "http://event.listener.url", ioutil.NopCloser(bytes.NewBuffer(reqBody)))
```

Comment:
nit: there are still two `http.NewRequest()` calls in this file, should we change them all to `httptest.NewRequest()`?

Reply:
`httptest` is useful for creating new incoming requests. The other two uses of `http.NewRequest()` are to create outgoing requests so I think we should keep them that way!

Reply:
Ah, that's what I get for skimming the code 😅 👍